obal/data/module_utils/obal.py

```diff
@@ -0,0 +1,28 @@
+"""
+Ansible module helper functions for obal modules
+"""
+import subprocess
+
+def get_changelog_evr(specfile):
+    """get the EVR from the last changelog entry in the specfile"""
+    evr = subprocess.check_output([
+        'rpm',
+        '--query',
+        '--changelog',
+        '--specfile',
+        specfile
+    ])
+    return evr.split("\n")[0].split(" ")[-1]
+
+def get_specfile_evr(specfile):
+    """get the EVR from the source header of the specfile"""
+    return subprocess.check_output([
+        'rpmspec',
+        '--query',
+        '--queryformat',
+        '%{evr}',
+        '--undefine',
```

Comment:
we should also undefine `foremandist`, @komidore64 want to do this in this PR, or in another?